include/z64message.h

```diff
@@ -15,25 +15,33 @@ struct OcarinaStaff;
 struct PlayState;
 
 typedef enum TextBoxType {
-    /* 0x00 */ TEXTBOX_TYPE_0,
-    /* 0x01 */ TEXTBOX_TYPE_1,
-    /* 0x02 */ TEXTBOX_TYPE_2,
-    /* 0x03 */ TEXTBOX_TYPE_3,
+    /* 0x00 */ TEXTBOX_TYPE_BLACK,
+    /* 0x01 */ TEXTBOX_TYPE_WOODEN,
+    /* 0x02 */ TEXTBOX_TYPE_BLUE_FADED,
+    /* 0x03 */ TEXTBOX_TYPE_OCARINA,
     /* 0x04 */ TEXTBOX_TYPE_4,
-    /* 0x05 */ TEXTBOX_TYPE_5,
-    /* 0x06 */ TEXTBOX_TYPE_6,
-    /* 0x07 */ TEXTBOX_TYPE_7,
-    /* 0x08 */ TEXTBOX_TYPE_8,
-    /* 0x09 */ TEXTBOX_TYPE_9,
+    /* 0x05 */ TEXTBOX_TYPE_CLEAR,
+    /* 0x06 */ TEXTBOX_TYPE_DISPLAY_ALL,
+    /* 0x07 */ TEXTBOX_TYPE_CLEAR_DISPLAY_ALL,
+    /* 0x08 */ TEXTBOX_TYPE_BLUE,
+    /* 0x09 */ TEXTBOX_TYPE_PAUSE_INFO,
     /* 0x0A */ TEXTBOX_TYPE_A,
     /* 0x0B */ TEXTBOX_TYPE_B,
-    /* 0x0C */ TEXTBOX_TYPE_C,
-    /* 0x0D */ TEXTBOX_TYPE_D,
-    /* 0x0E */ TEXTBOX_TYPE_E,
+    /* 0x0C */ TEXTBOX_TYPE_TITLE_CARD,
+    /* 0x0D */ TEXTBOX_TYPE_NOTEBOOK_NOTIFICATION,
+    /* 0x0E */ TEXTBOX_TYPE_OCARINA_FREE_PLAY,
     /* 0x0F */ TEXTBOX_TYPE_F,
     /* 0x10 */ TEXTBOX_TYPE_MAX
 } TextBoxType;
 
+#define TEXTBOX_BG_DEFAULT 0x0
+#define TEXTBOX_BG_WOODEN 0x1
+#define TEXTBOX_BG_OCARINA 0x2
+#define TEXTBOX_BG_FADED 0x3
+#define TEXTBOX_BG_NOTEBOOK 0x4
+
+#define TEXTBOX_BG_NONE 0xE
```

Comment:
I see you changed this from an enum to defines because the rom didn't match.
This is very likely to be caused because of a bss reorder.
Try changing this back to an enum, build and then run `tools/fix_bss.py`. This script _should_ fix the reordering. If it doesn't then ask for help so we can investigate the issue.

Reply:
OK, _now_ it is fixed. "Supposedly" was definitely the operative word.